doc/source/reference/arrays.dtypes.rst

```diff
@@ -225,16 +231,18 @@ Array-protocol type strings (see :ref:`arrays.interface`)
    supported kinds are
 
    ================   ========================
-   ``'b'``            boolean
+   ``'?'``            boolean
+   ``'b'``            (signed) byte
+   ``'B'``            unsigned byte
```

Comment:
Also, it seems `p` and `P` are things, which I guess correspond to `intp` and `uintp`? These have different meanings in `struct.pack`, which is potentially confusing